docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -722,9 +722,11 @@
           # This task is getting the last column as RECONCILED status value
           # TODO(ecandotti): Modify this block to not retrieve the last column, but instead recognize the column number for RECONCILED in each resource.
           # TODO(ecandotti): This will eliminate the need to always have the RECONCILED column as the last column when a new state is added.
-          - name: Retrieve kubectl resources reconciled status
+          # TODO(yuxing): Remove the monitor of the system resource for now, need to add it back and calculate the timeout
+          # based on the number of nodes and their inv status.
+          - name: Get not reconciled resource
             shell: >-
-              (kubectl -n deployment get datanetworks,platformnetworks,systems,ptpinstances,ptpinterfaces;
+              (kubectl -n deployment get datanetworks,platformnetworks,ptpinstances,ptpinterfaces;
```

Comment:
Since we are checking the last column, I think it's safer to list values for only `.status.reconciled`.
I mean something like this:
```
 (kubectl -n deployment get datanetworks,platformnetworks,ptpinstances,ptpinterfaces -o custom-columns=RECONCILED:.status.reconciled;
               kubectl -n deployment get hosts "{{ get_host_name.stdout}}" -o custom-columns=RECONCILED:.status.reconciled) |
               awk '$NF ~ /^false/ {print}'

```

Current implementation wouldn't work for PlatformNetwork since it's having deploymentScope as the last column instead:
https://github.com/Wind-River/cloud-platform-deployment-manager/blob/master/config/crd/bases/starlingx.windriver.com_platformnetworks.yaml#L38

Bootstrap configuration for oam/mgmt and admin is ignored anyway, so it wouldn't affect for the most part as of now.

Reply:
Thanks, updated as suggested, printed the resource name for the next tasks for debugging purpose